### PR TITLE
Implement Add book, assign author method

### PR DESCRIPTION
**Pull request by amexabee**
### The following options are available:
- List of games
- List all authors (e.g. 'Stephen King')
- Add a game

### CODE REVIEW

File: src/functions/assign_author.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  games.each_with_index do |game, index|
+    puts "#{index}) #{game.multiplayer} - #{game.publish_date}"
+  end
+  selected_index = gets.chomp.to_i
+  print 'Author first name: '
+  first_name = gets.chomp
+  print 'Author second name: '
+  second_name = gets.chomp
+  author = Author.new(first_name, second_name)
+  books[selected_index].add_author(author)
```
Comment (mperezbazan) [inline src/functions/assign_author.rb:54]: - [] Kindly check the add_author method when you assign an Author to a game. You have this error, so maybe you can correct this.
image

**Review by mperezbazan — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @amexabee 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Great implementation for the functions  :heavy_check_mark:👨🏻‍💻
- Great work using the right Gitflow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/functions/assign_author.rb (hunk 1 above)
Comment (amexabee) [reply src/functions/assign_author.rb]: Thank you! Done!

File: src/functions/list_authors.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def list_authors
+  puts 'List of Authors:'
+  puts 'No authors in the catalog' if authors.empty?
+  labels.each do |element|
+    puts "ID: #{element.id} - First Name: #{element.first_name} - Last Name: #{element.last_name}"
+  end
+end
```
Comment (mperezbazan) [inline src/functions/list_authors.rb:7]: - [] Kindly check when you list authors, the app doesn't show any message even if you create an author:
image

**Review by mperezbazan — CHANGES_REQUESTED** [review]
Hi @amexabee  , 

Great work on making the changes requested.
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Github flow 
✔️ PRO PR
✔️ No linter error 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/functions/list_authors.rb (hunk 2 above)
Comment (amexabee) [reply src/functions/list_authors.rb:7]: Thank you. I will fix it.

**Review by mperezbazan — APPROVED** [review]
Hi @amexabee 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**